### ISSUE TITLE
[RFC] Support for simple encrypted settings

### ISSUE THREAD

**Issue by AbdealiJK** [rochacbruno/dynaconf#428]
**Is your feature request related to a problem? Please describe.**
I think it would be useful if dynaconf can support its own simple secret encryption/decryption loader (probably powered by cryptography or pynacl). The main aim would be to support simple scenarios where users want to store their configs encrypted - but don't have a complex redis or dynaconf setup.

**Describe the solution you'd like**
 - A CLI command to create encrypted setting values
 - A builtin loader which can decrypt these encrypted values in a generic way
Potentially, it could be that the `.secrets.X` file can be assumed to have all encrypted configs ?

**Describe alternatives you've considered**
I'm considering currently implementing my own custom loader to do something similar. The custom loader functionality is pretty neat and shoudl allow for this (I'm still figuring it all out though)
But I am hoping here that this is a simple-ish loader to support, and would improve security with minimal effort for some users who are currently saving passwords in clear text files.

**Comment by rochacbruno** [comment]
We were considering #248 JOSE as a file format and also SOPS #224

**Comment by AbdealiJK** [comment]
Hi @rochacbruno Thanks for the reply!
I did see those issues, and I was considering them. Here are my comments on those:
 - **SOPS**: Seems like I need to install sops in my environment and the place where I use my application is quite restricted, and installing any new software is quite complex (kinda also why I avoided vault/redis)
 - **JOSE**: This seemed to be simpler (i.e. just a python library?) But I didnt go too much in detail here because I use the "python file" format (and not JSON, YAML, etc.). I know it's not recommended - but it provides some stuff that I care about :)

I was thinking of this something like - https://pypi.org/project/secureconfig/ (or atleast similar based on the README)
But that uses cryptography + configparser
